x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -74,6 +102,34 @@ export const Controls: FC<Props> = ({ analyticsId, modelId, details, getNodeData
   const nodeLabel = selectedNode?.data('label');
   const nodeType = selectedNode?.data('type');
 
+  const onCreateJobClick = useCallback(async () => {
+    const indexId = getIndexPatternIdFromName(nodeLabel);
+
+    if (indexId) {
+      await navigateToPath(`/data_frame_analytics/new_job?index=${encodeURIComponent(indexId)}`);
```

Comment:
Please use the URL generator instead.

Reply:
Good catch! Updated in 24e03b4ac632921bafc7dee7d4cec34c9b22104c